awesome_provider/docker-compose.yml

```diff
@@ -23,15 +23,15 @@ services:
         networks: 
             - backend
         volumes: 
-            - 'container_data:/docker-entrypoint-initdb.d'        
```

Comment:
I don't think we need to remove this volume, we use it to load a fresh instance of our DB.
See the documentation here: https://hub.docker.com/_/mysql -> Section: `Initializing a fresh instance`

Probably using two volumes is better, one for initializing our DB and another to persist data.